From 46141467794011ef8078f78998bece5c31094e4f Mon Sep 17 00:00:00 2001
From: Ionut Vicovan <Ionut.Vicovan@nxp.com>
Date: Tue, 2 Feb 2021 09:28:53 +0000
Subject: [PATCH] Fix build with gcc10.2

It fixes:
error: 'string' in namespace 'std' does not name a type

Signed-off-by: Ionut Vicovan <Ionut.Vicovan@nxp.com>
---
 interface/vsomeip/primitive_types.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/interface/vsomeip/primitive_types.hpp b/interface/vsomeip/primitive_types.hpp
index f0dd136..26a9fd7 100644
--- a/interface/vsomeip/primitive_types.hpp
+++ b/interface/vsomeip/primitive_types.hpp
@@ -8,6 +8,7 @@
 
 #include <array>
 #include <cstdint>
+#include <string>
 
 #ifndef _WIN32
 #include <sys/types.h>
-- 
2.17.1

